tests/ui_tests/common.py

```diff
@@ -262,6 +257,11 @@ def id(self) -> str:
     def fixtures_name(self) -> str:
         return f"{self.model}_{self.language}_{self.name}"
 
+    @classmethod
+    def from_fixtures(cls, fixtures_name: str, group: str) -> Self:
+        model, lang, name = fixtures_name.split("_", maxsplit=2)
+        return cls(model, group, name, lang)
```

Comment:
20 lines above, we create `cls` using keyword-arguments, maybe it makes sense to use kwargs here as well (with 4 args it is already easy to mistake the order - and the type-check does not help here, as all are strings)

Reply:
you're right, 0b99c699d880